docs/typescript.md

```diff
@@ -58,10 +61,10 @@ If your edit requires extra steps, please [file a bug](https://github.com/StackE
 
 Bug: Values passed to `CLI_DEFAULTS` (and the corresponding `-v` command-line option) don’t show up as global variables
 
-Workaround: create a new `.d.ts` file in the same folder as your `dnsconfig.js` file. In that file, add the following line for each variable you want to use. 
+Workaround: create a new `.d.ts` file in the same folder as your `dnsconfig.js` file. In that file, add the following line for each variable you want to use (replacing `VARIABLE_NAME` with the name of the variable. 
```

Comment:
Forgot a `)`, oops!

```suggestion
Workaround: create a new `.d.ts` file in the same folder as your `dnsconfig.js` file. In that file, add the following line for each variable you want to use (replacing `VARIABLE_NAME` with the name of the variable).
```

Reply:
Also, I don’t think literally naming the file `.d.ts` would work, I should have been more explicit here

```suggestion
Workaround: create a new file with the `.d.ts` extension in the same folder as your `dnsconfig.js` file (such as `variables.d.ts`, but the name doesn’t matter). In that file, add the following line for each variable you want to use (replacing `VARIABLE_NAME` with the name of the variable).
```